README.md

```diff
@@ -231,12 +231,12 @@ Examples to start the fictitious file named `foo_measurement.py`:
 
 ### Create Executable for Python Scripts
 
-To create an executable from a measurement, measurement authors can use the [pyinstaller](https://www.pyinstaller.org/) tooling. During the executable creation, the user can also embed the User Interface file using the `--add-data "<path_of_the_UI_File>;."`.
+To create an executable from a measurement, you can use the [pyinstaller](https://www.pyinstaller.org/) tooling. If you are using a virtual environment, install `pyinstaller` to your venv.
```

Comment:
"If you are using a Poetry project, add `pyinstaller` to its `dev-dependencies`."